app/controllers/identity.js

```diff
@@ -0,0 +1,126 @@
+import Controller from '@ember/controller';
+import { action } from '@ember/object';
+import { tracked } from '@glimmer/tracking';
+import ENV from 'website-my/config/environment';
+
+const BASE_URL = ENV.BASE_API_URL;
+
+export default class IdentityController extends Controller {
+  @tracked isEditClicked = false;
+  @tracked isGenerateChaincodeClicked = false;
+  @tracked identityURL = '';
```

Comment:
Why there are two identityURL?

Reply:
Oops, I must have forgotten to remove this one.